hazelcast-client-new/src/test/java/com/hazelcast/client/atomiclong/ClientAtomicLongTest.java

```diff
@@ -98,7 +99,7 @@ public void apply_whenException() {
         try {
             ref.apply(new FailingFunction());
             fail();
-        } catch (ExpectedRuntimeException expected) {
+        } catch (HazelcastException expected) {
```

Comment:
We are losing this kind of support via protocol. 
One suggestion is to give a way to register error codes and exceptions to user.